services/QuillLMS/app/workers/internal_tool/email_feedback_history_session_data_worker.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class InternalTool::EmailFeedbackHistorySessionDataWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: SidekiqQueue::LOW
+
+  def perform(activity_id, start_date, end_date, filter_type, responses_for_scoring, email)
+    feedback_histories = FeedbackHistory.session_data_for_csv({activity_id: activity_id, start_date: start_date, end_date: end_date, filter_type: filter_type, responses_for_scoring: responses_for_scoring})
```

Comment:
For readability, I like to break method calls with 3+ parameters into multiple lines:

```
 feedback_histories = FeedbackHistory.session_data_for_csv(
    activity_id: activity_id, 
    start_date: start_date, 
    end_date: end_date, 
    filter_type: filter_type, 
    responses_for_scoring: responses_for_scoring
)
```